homeassistant/components/sonos/favorites.py

```diff
@@ -131,6 +134,21 @@ def update_cache(self, soco: SoCo, update_id: int | None = None) -> bool:
             except SoCoException as ex:
                 # Skip unknown types
                 _LOGGER.error("Unhandled favorite '%s': %s", fav.title, ex)
+        for playlist in new_playlists:
+            playlist_reference = DidlFavorite(
+                title=playlist.title,
+                parent_id=playlist.parent_id,
+                item_id=playlist.item_id,
+                resources=playlist.resources,
+                desc=playlist.desc,
+            )
+            playlist_reference.reference = playlist
+            try:
+                self._favorites.append(playlist_reference)
+            except SoCoException as ex:
```

Comment:
I don't see how append can raise this exception. I do understand you were copying the above code block which does the same thing.

Reply:
Apparently codecov agrees, and I don't see how it could happen either. Is it the ```if fav.reference.resources:``` which can throw in the other code block? Oh well, I removed it so doesn't really matter any more.

Done.

Do I now leave these comments like this, unresolved, so you can validate/resolve your review?